cookbooks/web/attributes/default.rb

```diff
@@ -1,3 +1,6 @@
 default[:web][:base_directory] = "/srv/www.openstreetmap.org"
 default[:web][:pid_directory] = "/var/run/web"
 default[:web][:log_directory] = "/var/log/web"
+
+# need the APT source for cgimap to be able to install the package
+default[:apt][:sources] = node[:apt][:sources] | ["openstreetmap-cgimap"]
```

Comment:
Yes you could put it in the role. Seems I was wrong about having ones which did it in the recipe having now looked...